ddl/index.go

```diff
@@ -894,6 +894,8 @@ func (w *addIndexWorker) backfillIndexInTxn(handleRange reorgIndexTask) (taskCtx
 	return
 }
 
+var addIndexSpeedCounter = metrics.AddIndexTotalCounter.WithLabelValues("speed")
```

Comment:
IMHO this metric is not "speed", its a count, so the naming can be confusing

Reply:
em... any suggestion？